src/client/components/pages/Courses/NotesModal.tsx

```diff
@@ -166,42 +172,52 @@ const NotesModal: FunctionComponent<NotesModalProps> = function ({
                 placeholder="Some course notes"
                 isLabelVisible={false}
                 labelPosition={POSITION.TOP}
+                isDisabled={!isEditable}
                 onChange={
-                  ({ target: { value } }: ChangeEvent<HTMLTextAreaElement>) => {
-                    setIsChanged(true);
-                    setCourseNotes(value);
+                  ({
+                    target: { value, attributes },
+                  }: ChangeEvent<HTMLTextAreaElement>) => {
+                    const disabled = !!attributes.getNamedItem('disabled');
+                    if (!disabled) {
```

Comment:
Instead of the quadruple-negation here, what about using the `isEditable` value directly, e.g.

```suggestion
                    if (isEditable) {
```

That seems like it will be easier to parse to future maintenance, and ensures that we're always using the same value (i.e. the `isEditable` prop) to control each part of the modal

Reply:
I did think about doing this but my decision not to was based on attaching the blocking of the handler to the `disabled` prop. That way the handler fires or doesn't fire based on the presence of the field being `disabled` and it behaves more like you'd expect.

I alluded to this in the commit message of af7213bcdbb653d95fa63ad4549c261a67317f9d but I don't think I outlined that specifically.

Reply:
@jonseitz Addressed this in 515c415